### DISCUSSION TITLE
integrating IBM cloud object storage for tempo backend

### DISCUSSION THREAD

**Discussion by Rajarajan97** [grafana/tempo#3651] [General]
do we have any ongoing work for integrating IBM cloud object storage for tempo backend?
@joe-elliott  @electron0zero

**Comment by joe-elliott** [comment]
We have no plans to add direct support for IBM cloud object storage.

According to their [marketing page](https://www.ibm.com/products/cloud-object-storage) it is s3 compatible. It's possible Tempo will work out of the box using the s3 client.

**Comment by Rajarajan97** [comment]
yes it is compatible. Thank you
@joe-elliott
